addons/mail/static/src/utils/timer/tests/timer_tests.js

```diff
@@ -14,11 +14,12 @@ QUnit.module('timer_tests.js', {
         this.timers = [];
 
         this.start = async (params) => {
-            const { env, widget } = await start(Object.assign({}, params, {
+            const { env, widget, ...rest } = await start(Object.assign({}, params, {
```

Comment:
rest means repos, you probably meant remainder?

But what we typically do is "const res =" "return res", and deconstruct what is needed from res on another list. (see other tests, and keep a similar shape for easier removing them later)

Reply:
according to the Cambridge dictionary, rest also means : `the other things, people, or parts that remain or that have not been mentioned`. Anyway, I just changed it in order to keep a similar shape for all tests as you suggested :+1:

Reply:
Fair enough, still it's a bit strange to use `rest` as a word out of context due to its multiple meanings (and the main one being the one I said). It would be ok if it was with more context (...the rest of ...).

Not specific to this word by the way, just in general it's better to avoid single words with multiple meanings.

But of course it's a non issue if you change the shape.